TLS/src/main/java/de/rub/nds/tlsattacker/dtls/protocol/handshake/HelloVerifyRequestHandler.java

```diff
@@ -23,12 +23,13 @@
  * @author Florian Pfützenreuter <florian.pfuetzenreuter@rub.de>
  * @param <HandshakeMessage>
  */
-public class HelloVerifyRequestHandler<HandshakeMessage extends HelloVerifyRequestMessage> extends
```

Comment:
The Parameter for this Generic Type argument was renamed as there is a type HandshakeMessage which is hidden through this parameter of the same name. Thus it wasn't really clear which type (the parameterized one or HandshakeMessage) was ment when HandshakeMessage was written in the code. Renaming the type argument resolves this disambiguation.

The change in the constructor are due to the field "correctProtocolMessageClass" being of Class<? extends Message>; and thus needing typecast to take in the Class<Object> returned by HelloVerifyRequestMessage.class